packages/datadog-plugin-amqp10/src/consumer.js

```diff
@@ -11,8 +11,8 @@ class Amqp10ConsumerPlugin extends ConsumerPlugin {
     const source = getShortName(link)
     const address = getAddress(link)
 
-    this.startSpan('amqp.receive', {
-      service: this.config.service || `${this.tracer._service}-amqp`,
+    this.startSpan(this.operationName(), {
+      service: this.config.service || this.serviceName({ service: this.tracer._service }),
```

Comment:
Similarly, isn't this a global fallback?

Reply:
Indeed, same as above.